### ISSUE TITLE
ResourceViewSettings size

### ISSUE THREAD

**Issue by Cedricoh** [syncfusion/flutter-widgets#382]
Hi,

ResourceViewSettings.size should be separated into height and width, to allow more customization.

Thanks !

**Comment by muthulakshmi-123** [comment]
Hi,

Thank you for contacting Syncfusion support. 

Based on provided information your requirement is "Can we give separate height and width for size of resourceViewSettings?” As per the current implementation of the calendar, the [size](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/ResourceViewSettings/size.html) property of [resourceViewSettings](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/ResourceViewSettings-class.html) indicates the width of the resource panel in the timeline views. But you can customize the number of visible resource count in the timeline views. Kindly, share more details about your requirement. It will be helpful for us to analyze and resolve your requirement. 

Please refer to the following UG link for more information.  
https://help.syncfusion.com/flutter/calendar/resource-view

We hope that this helps you. Please let us know if you need further assistance.

Regards,
Muthulakshmi K

**Comment by Cedricoh** [comment]
Yes, but I would like to be able to specify the height if the resource panel (and thus the entire row).

**Comment by muthulakshmi-123** [comment]
Hi,

Thanks for the update.

Based on provided information your requirement is **"Can we set height for resource item?”** As per the current implementation of the calendar, there is no direct support to customize the height for the separate resource item. But you can set the [visibleResourceCount](https://pub.dev/documentation/syncfusion_flutter_calendar/latest/calendar/ResourceViewSettings/visibleResourceCount.html), which will split the available viewport size to each resource item based on the count set to this property

Please refer to the following UG link for more information.  
https://help.syncfusion.com/flutter/calendar/resource-view#visible-resource-count

We hope that this helps you. Please let us know if you need further assistance.

Regards,
Muthulakshmi K

**Comment by TheCarpetMerchant** [comment]
I made a feature request. I'm closing the issue, thanks for the help !
